app/app_client.go

```diff
@@ -1075,7 +1082,7 @@ func (c *AppClient) UpdateModule(
 		Description: description,
 		Models:      pbModels,
 		Entrypoint:  entrypoint,
-		FirstRun:    opts.firstRun,
+		FirstRun:    opts.FirstRun,
```

Comment:
See above re nil pointer dereference.

Reply:
We're not dereferencing something here, so wouldn't it just pass in a `nil`? `FirstRun` is a `*string` type.

Reply:
Ah yes, never mind. I get it now!